docs/readmes/orc8r/deploy_build.md

```diff
@@ -76,32 +83,31 @@ We'll build the Orchestrator Helm charts, as well as publish them to a
 To start, create a private GitHub repo to use as your Helm chart repo. We'll
 refer to this as `GITHUB_REPO`.
 
-Next, package the Magma Helm charts and publish them to the GitHub
-repo. You'll have to check out a temporary commit to build the `1.4.36` version
-of the orc8r chart for 1.3.x because `1.4.37` on the head of the release
-branch has some changes that you shouldn't try to deploy yet.
+Next, define some necessary variables
 
 ```bash
-cd MAGMA_ROOT
-git checkout a7580153
+export GITHUB_REPO=<github_repo>
+export GITHUB_REPO_URL=<github_repo_url>
+export GITHUB_USERNAME=<github_username>
+export GITHUB_ACCESS_TOKEN=<github_access_token>
```

Comment:
Add a note above with this link, to head off any questions: https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token

Reply:
This is included in the prerequisites section